core/services/workflows/v2/engine.go

```diff
@@ -336,11 +346,34 @@ func (e *Engine) CallCapability(ctx context.Context, request *sdkpb.CapabilityRe
 		},
 	}
 
+	meterReport, ok := e.meterReports.Get(request.ExecutionId)
+	if !ok {
+		e.cfg.Lggr.Error("no metering report found for %v", request.ExecutionId)
+	}
+	capInfo, err := capability.Info(ctx)
+	if err != nil {
+		e.cfg.Lggr.Error("could not get capability info for %v", capReq.CapabilityId)
+	}
+	// TODO: After (CAPPL-881) replace with call ID
+	count := meterReport.IncrementRefCount(metering.ReportStepRef(capReq.CapabilityId))
+	callID := capReq.CapabilityId + strconv.FormatUint(count, 10)
+	ref := metering.ReportStepRef(callID)
+	err = meterReport.ReserveStep(ref, capInfo)
+	if err != nil {
+		e.cfg.Lggr.Error("could not reserve for %s: %w", callID, err)
```

Comment:
Is this fail-open on purpose? If so, maybe put it behind a config flag?

Reply:
Yes, everything is to be fail open. We will harden this up in subsequent PRs by having some places where we can't continue.